src/commonMain/kotlin/operations/numeric/Multiplication.kt

```diff
@@ -0,0 +1,23 @@
+package operations.numeric
+
+import JsonLogicException
+import LogicOperation
+import asList
+import operations.numeric.unwrap.StrictUnwrapStrategy
+
+internal object Multiplication : LogicOperation, DoubleTypeSensitiveOperation, StrictUnwrapStrategy {
+    override val key: String = "*"
+
+    override fun invoke(expression: Any?, data: Any?): Any? {
+        val values = expression.asList
+        return when (values.size) {
+            0 -> throw JsonLogicException("")
```

Comment:
inconsistent, subtraction returns null in that case (but I guess it will be sorted out in the separate subtask, right?)

Reply:
In the JS impl it throws some internal js operation exception so I decided to return null 
<img width="638" alt="image" src="https://user-images.githubusercontent.com/84186160/153838399-6c27e92a-3166-4ee2-97c5-26b64b58c473.png">